sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_sas_builder.hpp

```diff
@@ -90,6 +90,11 @@ namespace Azure { namespace Storage { namespace Sas {
      */
     SetImmutabilityPolicy = 256,
```

Comment:
In general, I'm not a fan of expressing bit values in decimal. It tends to be more error prone than `0x100`, `0x200`, `0x400`, etc.

Reply:
I feel decimal is more intuitive. I can tell 256 is twice as 128 at one glance. But I need to pause for a moment to tell 0x100 is twice as 0x80.

Reply:
Ok, I'm confused then. From the values, this looked like [flags enumeration](https://softwareengineering.stackexchange.com/questions/194412/using-scoped-enums-for-bit-flags-in-c) (where you would perform logical operators ("or", "and", "xor", etc) to values). 

If you are applying logical operators to these values, then the only thing that matters is their bit position. And if the only thing you care about is their bit position, then representing the values in hex makes much more sense than decimal because it's trivial to determine the bit positions of a hex digit.

However, if there are no circumstances where you would apply logical operators on the values, then using numeric values makes sense.